src/LibraryViewExtension/Handlers/NodeItemDataProvider.cs

```diff
@@ -25,18 +25,22 @@ class LoadedTypeItem
         public string keywords { get; set; }
     }
 
-    class LoadedTypeData
+    class LoadedTypeData<T>
```

Comment:
I am wondering if it would have worked as it is when loadedTypes internally contains the list of LoadedTypeItemExtended?